easybuild/easyconfigs/p/Python/Python-3.8.2-GCCcore-9.3.0.eb

```diff
@@ -20,6 +20,7 @@ dependencies = [
     ('ncurses', '6.2'),
     ('SQLite', '3.31.1'),
     ('XZ', '5.2.5'),
+    ('UnZip', '6.0'),
```

Comment:
@terjekv This is only a build dep, or am I overlooking something?

See also #10427

Reply:
Correct, my bad.